deploy/helm/llm-proxy/README.md

```diff
@@ -9,14 +9,66 @@ This Helm chart deploys the LLM Proxy server to Kubernetes.
 
 ## Installation
 
-**Important:** The application requires a `MANAGEMENT_TOKEN` environment variable to be configured. This minimal chart does not yet include secret management. You must manually provide `MANAGEMENT_TOKEN` (for example, via a Kubernetes Secret and corresponding pod configuration) for the deployment to function correctly. Secret handling support will be added in issue #203.
+### Required: Secret Management
+
+The application requires a `MANAGEMENT_TOKEN` environment variable for admin operations. This chart supports two approaches:
```

Comment:
Docs say MANAGEMENT_TOKEN is required (true), but the chart doesn’t enforce it. Either fail fast in templates or explicitly call out that Helm will install but the pod will crash without the secret wired.